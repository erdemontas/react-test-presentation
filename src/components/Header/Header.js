
import React from 'react';
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';
import { makeStyles } from '@material-ui/core/styles';
import { colors } from '@mui/material';

const useStyles = makeStyles({
    toolbar: {
        textAlign: "center",
        height: 80,
        background : 'red'
    },

    heading:{
        margin:"auto"
    }
});


const Header = () => {
    const classes = useStyles();
    return(
        <AppBar position="static" >
            <Toolbar className={classes.toolbar}>
                <Typography className={classes.heading} variant="h6" align="center">Todo App</Typography>
            </Toolbar>
        </AppBar>
    )
}

export default Header;