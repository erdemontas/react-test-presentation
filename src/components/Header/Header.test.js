import Header from "./Header"
import renderer from 'react-test-renderer';


describe("Snapshot test on Header Component", () => {
    test('render Header correctly', () => {
        const tree = renderer.create(<Header/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})