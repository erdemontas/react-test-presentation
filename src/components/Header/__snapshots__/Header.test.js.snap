// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test on Header Component render Header correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular makeStyles-toolbar-1 css-1t29gy6-MuiToolbar-root"
  >
    <h5
      className="MuiTypography-root MuiTypography-h5 MuiTypography-alignCenter makeStyles-heading-2 css-ph2xsj-MuiTypography-root"
    >
      Todo App
    </h5>
  </div>
</header>
`;
